git-sync/main.go

```diff
@@ -213,3 +221,27 @@ func setupGitAuth(username, password, gitURL string) error {
 
 	return nil
 }
+
+func setupGitSSH() error {
+	log.Println("setting up git SSH credentials")
```

Comment:
The existing code has lowercase "git" throughout, so I went with this for consistency's sake. I could replace them all though if you think that's better.